code-examples/quarkus-kafka-quickstart/src/main/resources/application.properties

```diff
@@ -26,9 +26,9 @@ quarkus.container-image.push=false
 
 %dev.kafka.sasl.mechanism=OAUTHBEARER
 %dev.kafka.sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required \
-  oauth.client.id="${RHOAS_CLIENT_ID}" \
-  oauth.client.secret="${RHOAS_CLIENT_SECRET}" \
-  oauth.token.endpoint.uri="${RHOAS_OAUTH_TOKEN_URL}" ;
+  oauth.client.id="${RHOAS_SERVICE_ACCOUNT_CLIENT_ID}" \
```

Comment:
So values we have been providing in context were totally different than values in the service account file?

Reply:
I like new variable names anyway :D

Reply:
Yes those were different.